changed wording (#74722)
ansible/ansible: README.rst

Ansible is a radically simple IT automation system. It handles
configuration management, application deployment, cloud provisioning,
ad-hoc task execution, network automation, and multi-node orchestration. Ansible makes complex

> changes like zero-downtime rolling updates with load balancers easy. More information on the ansible `website <https://ansible.com/>`_.

Reviewer: We do like to capitalize the name, unless we're referring directly to the repo or the CLI command. 
```suggestion
changes like zero-downtime rolling updates with load balancers easy. More information on the Ansible `website <https://ansible.com/>`_.
```

Design Principles
=================

> *  Have a extremly simple setup process with a minimal learning curve.

Reviewer: ```suggestion
*  Have an extremely simple setup process with a minimal learning curve.
```

> *  Manage machines fast and in parallel.

Reviewer: Fast is an adjective, quickly is an adverb, we need an adverb here.
```suggestion
*  Manage machines quickly and in parallel.
```
